app-api/src/main/resources/liquibase-sender.xml

```diff
@@ -250,5 +250,69 @@
             </column>
         </addColumn>
     </changeSet>
+    
+    <changeSet author="lpinto" id="20220210-1050">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="sender_sync_message" />
+            </not>
+        </preConditions>
+        <comment>Adding sender_sync_message table</comment>
 
+        <createTable tableName="sender_sync_message">
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" />
+            </column>
+            <column name="table_name" type="VARCHAR(100)">
+                <constraints nullable="false" />
+            </column>
+            <column name="identifier" type="VARCHAR(255)">
+                <constraints nullable="false" />
+            </column>
+            <column name="operation" type="VARCHAR(1)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="message_uuid" type="VARCHAR(38)">
+                <constraints nullable="false" unique="true"/>
+            </column>
+            <column name="request_uuid" type="VARCHAR(38)" />
+            <column name="date_created" type="DATETIME(3)">
+                <constraints nullable="false" />
+            </column>
+            <column name="snapshot" type="BOOLEAN">
+                <constraints nullable="false" />
+            </column>
+            <column name="status" type="VARCHAR(20)">
```

Comment:
I know am being picky here, apologies! But by unwritten convention we have been using VARCHAR(50) for these status columns, let's try to keep it uniform

Reply:
Alright, I'll take more attention to look for the existing column definitions when creating similar ones.
Changed to 50.